### ISSUE TITLE
Footnote anchor references are malformed

### ISSUE THREAD

**Issue by djw-m** [EnterpriseDB/docs#4415]
Footnotes should be linkable but in a recent doc, the foot noted item rendered in HTML as

```html
<td>2.3.0<sup id="fnref-a"><a data-gatsby-link="true" class="footnote-ref" href="/docs/pgd/4/rel_notes/#fn-a">a</a></sup></td>
```

while the target text was

```html
<p><sup id="fnref-a"><a data-gatsby-link="true" class="footnote-ref" href="/docs/pgd/4/rel_notes/#fn-a">a</a></sup> We released a patch to HARP 2.2.3 with bug fixes and enhancements. If you are using 2.2.3 or earlier, we recommend that you upgrade to 2.3.0.</p>
```

There's two identical id's and both have a superscript section that links to #fn-a, the wrong name afaict, so the hyperlink will go nowhere. 

Ideally figure out why, and fix it so footnotes are first class markup.

**Comment by josh-heyer** [comment]
When you get a chance @djw-m please add a link to a test case for this - it doesn't happen in *all* cases

**Comment by djw-m** [comment]
Tried to build a test case and failed.

What I did learn is that if you try and use footnotes repeatedly, you will get multiple repeated ids as footnotes does not like more¹ than¹ one¹ entry for a foot note - it generates source id links which are all the same and then has a single back link for them.

As this originally happened in a table where multiple things where being tagged with a footnote, I think we can conclude that t was this that was generating the oddlinks, not the footnotes alone. 

Propose adding guidance to the docs that footnotes are a singular mark to a singular text in the footer (and should not be used to label multiple items/texts).

Closing this one out of shame.
